### DISCUSSION TITLE
看视频点击作者个人主页，个人主页弹框无法移动

### DISCUSSION THREAD

**Discussion by moyv999** [Richasy/Bili.Uwp#234] [问答]
看视频点击作者个人主页，个人主页弹框无法移动，影响视频观看

**Comment by Richasy** [comment]
总是要找个位置放的……我不太相信有人能即看主页视频又看正在播放的视频，如果想看视频，就关掉弹框，如果想查看个人主页，就专心看主页就好

**Reply by X0980** [reply]
> 总是要找个位置放的......我不太相信有人能即看主页视频又看正在播放的视频，如果想看视频，就关掉弹框，如果想查看个人主页，就专心看主页就好

希望把up主页变成全屏，现在和1.0一样，只是把侧边改成了中间，内容太少，不方便查看

**Reply by cnoim** [reply]
> 总是要找个位置放的……我不太相信有人能即看主页视频又看正在播放的视频，如果想看视频，就关掉弹框，如果想查看个人主页，就专心看主页就好

电脑上出现这种小窗口感觉有点怪怪的，希望能加上一个开关让用户自行选择
